Foundation/NSValue.swift

```diff
@@ -96,7 +96,7 @@ public class NSValue : NSObject, NSCopying, NSSecureCoding, NSCoding {
         return NSSpecialValue._typeFromObjCType(type) != nil
     }
     
-    public convenience required init(bytes value: UnsafePointer<Void>, objCType type: UnsafePointer<Int8>) {
+    public convenience required init(bytes value: UnsafePointer<Void>?, objCType type: UnsafePointer<Int8>) {
```

Comment:
I'm not very happy with this. Do we care about zero-byte types enough to make this initializer optional?